.github/workflows/ci.yml

```diff
@@ -51,11 +51,8 @@ jobs:
       - name: Build Module
         run: yarn build
 
-      - name: Build Fixture
-        run: yarn nuxt build test/fixture
-
       - name: Tests
-        run: yarn jest
+        run: yarn jest --runInBand
```

Comment:
Was there anything in this PR causing issues in CI? We'd want to keep the CI workflow the same.

Reply:
This comment was marked resolved, but the change is still there and the comment is unanswered?